jobs/loc/TranslationsImportExport.yml

```diff
@@ -0,0 +1,56 @@
+# ==================================================================================
+# Pipeline for VsCodeExtension-Localization build definition
+# Runs OneLocBuild task to localize xlf file
+# ==================================================================================
+
+resources:
+  repositories:
+  - repository: self
+    clean: true
+
+trigger: none
+pr: none
+schedules:
+- cron: "0 7 * * *"
+  displayName: Daily 7 AM
+  branches:
+    include:
+    - main
+    - localization
+  always: true
+
+pool:
+  vmImage: 'windows-latest'
+
+steps:
+- task: CmdLine@2
+  inputs:
+    script: 'yarn install'
+
+- task: CmdLine@2
+  inputs:
+    script: 'yarn run translations-export'
+
+- task: OneLocBuild@2
+  env:
+    SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+  inputs:
+    locProj: 'jobs/loc/LocProject.json'
+    outDir: '$(Build.ArtifactStagingDirectory)'
+    isCreatePrSelected: false
+    prSourceBranchPrefix: 'locfiles'
+    packageSourceAuth: 'patAuth'
+    patVariable: '$(OneLocBuildPat)'
+    LclSource: lclFilesfromPackage
+    LclPackageId: 'LCL-JUNO-PROD-VMAKEFILE'
+    lsBuildXLocPackageVersion: '7.0.30510'
+
+- task: CmdLine@2
+  inputs:
+    script: 'node ./translations_auto_pr.js microsoft vscode-makefile-tools csigs $(csigsPat) csigs csigs@users.noreply.github.com "$(Build.ArtifactStagingDirectory)/loc" vscode-extensions-localization-export/vscode-extensions'
```

Comment:
I don't recall, but you might need to provide the csigs account with permission to push a branch and post a PR.